internal/operands/common-instancetypes/reconcile_test.go

```diff
@@ -251,6 +252,56 @@ var _ = Describe("Common-Instancetypes operand", func() {
 		ExpectResourceExists(preference, request)
 	})
 
+	It("should ignore virt-operator owned objects during reconcile when also provided by bundle", func() {
+		_, err = operand.Reconcile(&request)
+		Expect(err).ToNot(HaveOccurred())
+
+		instancetypeList := &instancetypev1beta1.VirtualMachineClusterInstancetypeList{}
+		Expect(request.Client.List(request.Context, instancetypeList, &client.ListOptions{})).To(Succeed())
+		Expect(len(instancetypeList.Items) > 0).To(BeTrue())
+
+		preferenceList := &instancetypev1beta1.VirtualMachineClusterPreferenceList{}
+		Expect(request.Client.List(request.Context, preferenceList, &client.ListOptions{})).To(Succeed())
+		Expect(len(preferenceList.Items) > 0).To(BeTrue())
+
+		// Mutate the instance type while also adding the labels for virt-operator
+		instancetypeToUpdate := instancetypeList.Items[0]
+		updatedCPUGuestCount := instancetypeToUpdate.Spec.CPU.Guest + 1
+		instancetypeToUpdate.Spec.CPU.Guest = updatedCPUGuestCount
+		instancetypeToUpdate.Labels = map[string]string{
+			virtv1.ManagedByLabel: virtv1.ManagedByLabelOperatorValue,
+		}
+		Expect(request.Client.Update(request.Context, &instancetypeToUpdate, &client.UpdateOptions{})).To(Succeed())
+
+		// Mutate the preference while also adding the labels for virt-operator
+		preferenceToUpdate := preferenceList.Items[0]
+		updatedPreferredCPUTopology := instancetypev1beta1.PreferCores
+		updatedPreferenceCPU := &instancetypev1beta1.CPUPreferences{
+			PreferredCPUTopology: &updatedPreferredCPUTopology,
+		}
+		preferenceToUpdate.Spec.CPU = updatedPreferenceCPU
+		preferenceToUpdate.Labels = map[string]string{
+			virtv1.ManagedByLabel: virtv1.ManagedByLabelOperatorValue,
+		}
+		Expect(request.Client.Update(request.Context, &preferenceToUpdate, &client.UpdateOptions{})).To(Succeed())
+
+		results, err := operand.Reconcile(&request)
+		Expect(err).ToNot(HaveOccurred())
+
+		// Assert that we have reported ignoring the attempt to reconcile the objects owned by virt-operator
+		for _, res := range results {
+			if res.Resource.GetName() == instancetypeToUpdate.Name || res.Resource.GetName() == preferenceToUpdate.Name {
+				Expect(res.OperationResult).To(Equal(common.OperationResultIgnored))
+			}
+		}
```

Comment:
Can you instead check if the `InstanceType` and `Preferece` with the label were not reverted?

Reply:
That is already done below.